charts/estafette-k8s-hpa-scaler/templates/_helpers.tpl

```diff
@@ -0,0 +1,9 @@
+{{- define "labels" }}
+labels:
+  app: {{.Chart.Name | quote}}
+  version: {{.Chart.Version | quote}}
+  appVersion: {{.Chart.AppVersion | quote}}
```

Comment:
What's the difference between `version` and `appVersion`?

Reply:
https://helm.sh/docs/developing_charts/#the-appversion-field
`version` is the version of the chart. And `appVersion` is the version of the image being used.